src/rpc/server/pageDispatcher.ts

```diff
@@ -83,6 +86,9 @@ export class PageDispatcher extends Dispatcher implements PageChannel {
   }
 
   async waitForEvent(params: { event: string }): Promise<any> {
+    const result = await this._page.waitForEvent(params.event);
+    if (result instanceof ConsoleMessage)
```

Comment:
Might as well do Request, Response and Frame.

Reply:
Done